packages/api/AlvTime.Persistence/Repositories/FlexhourStorage.cs

```diff
@@ -329,27 +323,31 @@ public PaidOvertimeEntry RegisterPaidOvertime(GenericHourEntry request, int user
         throw new ValidationException("Not enough available hours");
     }
 
+
+
     public PaidOvertimeEntry CancelPayout(int userId, int id)
     {
-        var payout = _context.PaidOvertime.FirstOrDefault(po => po.Id == id && po.User == userId);
+        var paidOvertime = _context.PaidOvertime.FirstOrDefault(po => po.Id == id && po.User == userId);
 
-        if (!CanBeDeleted(payout))
+        if (!CanBeDeleted(paidOvertime))
         {
             throw new ValidationException("Selected payout must be latest ordered payout");
         }
 
-        if (payout != null && payout.Date.Month >= DateTime.Now.Month && payout.Date.Year == DateTime.Now.Year)
+        if (paidOvertime != null && paidOvertime.Date.Month >= DateTime.Now.Month && paidOvertime.Date.Year == DateTime.Now.Year)
         {
-            _context.PaidOvertime.Remove(payout);
+            _context.PaidOvertime.Remove(paidOvertime);
             _context.SaveChanges();
 
+            _salaryService.DeleteOvertimePayout(userId, paidOvertime.Id);
+

```

Comment:
Generelt spørsmål: hva er konsekvensen av å ha foreign keys på tvers av databaser? Altså for eksempel at UserId i økonomidatabasen peker på en bruker i den "vanlige" databasen. Kan vi havne i en situasjon hvor dette kan skape problemer? Aner ikke selv, kanskje bare noe å ha i bakhodet 😃

Reply:
Usikker på konsekvenser. Det er jo ingen direkte kobling :/ Tror kanskje vi bare må holde et øye med det?